Tests.Integration.Presentation.Web/ItSystem/ItSystemUsedByTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using Core.DomainModel.ItSystemUsage;
+using Core.DomainModel.Organization;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Integration.Presentation.Web.Tools.Model;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.ItSystem
+{
+    public class ItSystemUsedByTests : WithAutoFixture
+    {
+
+        [Theory]
+        [InlineData(OrganizationRole.GlobalAdmin)]
+        [InlineData(OrganizationRole.User)]
+        public async Task Api_User_Can_Get_Organizations_That_Use_An_It_System(OrganizationRole apiUserType)
+        {
+            //Arrange
+            var user = TestEnvironment.GetCredentials(apiUserType, true);
+            var loginDto = ObjectCreateHelper.MakeSimpleLoginDto(user.Username, user.Password);
+            var token = await HttpApi.GetTokenAsync(loginDto);
```

Comment:
I de fleste tests er man ligeglad med user og loginDto, men vil bare have en token for en bruger med en bestemt rolle. Lav en GetTokenAsync der tager en role og som laver de to linjer ovenfor samt kaldet til GetToken. Så sparer vi en del linjer